pkgs/applications/misc/blender/default.nix

```
@@ -63,6 +65,7 @@ stdenv.mkDerivation rec {
     ++ optional cudaSupport cudatoolkit
     ++ optional colladaSupport opencollada
     ++ optional spaceNavSupport libspnav;
+  propagatedBuildInputs = with python3Packages; [ numpy requests ];
```

```suggestion
  pythonPath = with python3Packages; [ numpy requests ];
```
I believe that's the proper attribute since blender is not a python library.  https://github.com/NixOS/nixpkgs/issues/118410#issuecomment-813017540

---

Thanks @gebner, I've changed accordingly.